kubert/src/log.rs

```diff
@@ -16,12 +23,129 @@ pub enum LogFormat {
     Json,
 }
 
+/// Configures the global default tracing filters.
+///
+/// A cloneable version of [`tracing_subscriber::EnvFilter`].
```

Comment:
fwiw, it would probably be fine to add a `Clone` impl upstream...

Reply:
I'm not sure if that makes sense. `EnvFilter` holds some `RwLock`s, so we'd either have to `Arc` internally or... We could impl Layer/Filter for Arc<T>? Which means `Arc<EnvFilter>` would Just Work.

Reply:
The filter holds internal state, but that state is only relevant when it's actively in use as part of a `Subscriber`. I think a `Clone` impl that creates a "fresh" `EnvFilter` with empty state would be reasonable to add, as it would be rebuilt if that filter instance was actually used in a `Subscriber`. 

The main reason I think this is probably worth doing upstream is that I've seen a number of projects use `EnvFilter` as a `clap` argument, so the transition to `clap` v4 is going to be painful for more than _just_ `kubert`...

Unfortunately, we can't currently add an impl for `Arc<T: Layer<S>>` without a breaking change, as the `Layer::on_layer` method takes an `&mut self` receiver. Forwarding this method isn't necessary for `EnvFilter`, but it _is_ necessary when a `Layer` is a set of nested layers that includes a `Filtered` layer. We can't add a general `impl<T, S> Layer for Arc<T> where T: Layer<S>` upstream unless we change that method's receiver to `&self`...which is being considered for the next `tracing-subscriber` breaking change (https://github.com/tokio-rs/tracing/issues/2101).